components/styled-components.js

```diff
@@ -11,27 +12,34 @@ export const CenteredContainer = styled.div`
 `;
 
 
+export const Progress = styled.p`
+  font-size: 40px; /* Ändere die Schriftgröße hier */
+  font-weight: bold; /* Ändere das Schriftgewicht hier */
+  margin-top: 10px; /* Hinzugefügter Abstand oben */
```

Comment:
Remove comments when you're done playing around with each of the css attributes.